packages/common/src/ui/layout/tables/components/Cells/NumberInputCell/NonZeroNumberInputCell.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { CellProps } from '../../../columns';
 import { BasicTextInput } from '@common/components';
-import { DomainObject } from '@common/types';
+import { RecordWithId } from '@common/types';
 import { useBufferState, useDebounceCallback } from '@common/hooks';
 import { NumUtils } from 'packages/common/src/utils/NumUtils';
 
-export const NonZeroNumberInputCell = <T extends DomainObject>({
+export const NonZeroNumberInputCell = <T extends RecordWithId>({
```

Comment:
ah, thanks. had also done in the stocktake PR - let's see who gets merged first 😁

Reply:
You win 😁